<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsSearchWidgetIfClass</class>
 <widget class="QWidget" name="CwmsSearchWidgetIfClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CwmsSearchWidgetIf</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_pqgbSearch">
     <property name="title">
      <string>Suche</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="m_pqrbLogicAnd">
        <property name="text">
         <string>Alle Suchkritererien müssen erfüllt sein</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pqrbLogicOr">
        <property name="text">
         <string>Mindestens ein Suchkriterium muss erfüllt sein</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="m_pqScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>500</width>
           <height>303</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_pqpbSearch">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wmsgui.qrc">
         <normaloff>:/icons/print-preview.png</normaloff>:/icons/print-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="wmsgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbSearch</sender>
   <signal>clicked()</signal>
   <receiver>CwmsSearchWidgetIfClass</receiver>
   <slot>SearchClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SearchClickedSlot()</slot>
 </slots>
</ui>
